service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/DraftOrdersControllerTest.java

```diff
@@ -380,15 +382,28 @@ void aboutToSubmitShouldReturnErrorsWhenNoHearingDetailsExistsForSealedOrder() {
                     .data(createCaseDataMap(directionWithShowHideValuesRemoved)
                         .put("standardDirectionOrder", order)
                         .put("judgeAndLegalAdvisor", JudgeAndLegalAdvisor.builder().build())
+                        .put("allocatedJudge", AllocatedJudge.builder().build())
                         .build())
                     .build())
                 .build();
 
             AboutToStartOrSubmitCallbackResponse response = postAboutToSubmitEvent(request);
 
             assertThat(response.getErrors())
-                .containsOnly("This standard directions order does not have a hearing associated with it. "
-                    + "Please enter a hearing date and resubmit the SDO");
+                .containsOnly("You need to enter a hearing date.");
```

Comment:
is this correct? have we updated error messages?

Reply:
Yes I spoke to Colette and mentioned to her about the previous error message and she asked me to change both to:

You need to enter the allocated judge.
You need to enter a hearing date.

Put the comment in the jira ticket